lib/minify.js

```diff
@@ -46,6 +47,7 @@ function minify(files, options) {
             wrap: false,
         }, true);
         set_shorthand("ie8", options, [ "compress", "mangle", "output" ]);
+        set_shorthand("safari10", options, [ "mangle" ]);
```

Comment:
In fact, why should it even be a toplevel option, if only `mangle` is affected?

Just keep it under `options.mangle.safari10` for now. If it turns out Safari becomes the next IE6 in terms of quirks, we can add a toplevel option then. Right now it just add noise to the global namespace.